lib/src/test/java/io/ably/lib/test/realtime/RealtimeAuthTest.java

```diff
@@ -192,6 +205,9 @@ public void auth_client_match_token_null_clientId() {
         }
     }
 
+    private void assertNotNull(String expected_token_value, String token) {
+    }
```

Comment:
What is the purpose of this method? :thinking:

Reply:
thanks, nice catch! it got here by a mistake